wzdx/sample_files/enhanced/icone/incidents_extended.json

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
Same here, is this one being used? I don't see a reference anywhere to it.

Reply:
removed